node/status_node_services.go

```diff
@@ -311,27 +311,36 @@ func (b *StatusNode) wakuService(wakuCfg *params.WakuConfig, clusterCfg *params.
 func (b *StatusNode) wakuV2Service(nodeConfig *params.NodeConfig, telemetryServerURL string) (*wakuv2.Waku, error) {
 	if b.wakuV2Srvc == nil {
 		cfg := &wakuv2.Config{
-			MaxMessageSize:          wakucommon.DefaultMaxMessageSize,
-			Host:                    nodeConfig.WakuV2Config.Host,
-			Port:                    nodeConfig.WakuV2Config.Port,
-			LightClient:             nodeConfig.WakuV2Config.LightClient,
-			KeepAliveInterval:       nodeConfig.WakuV2Config.KeepAliveInterval,
-			Rendezvous:              nodeConfig.Rendezvous,
-			WakuNodes:               nodeConfig.ClusterConfig.WakuNodes,
-			PeerExchange:            nodeConfig.WakuV2Config.PeerExchange,
-			EnableStore:             nodeConfig.WakuV2Config.EnableStore,
-			StoreCapacity:           nodeConfig.WakuV2Config.StoreCapacity,
-			StoreSeconds:            nodeConfig.WakuV2Config.StoreSeconds,
-			DiscoveryLimit:          nodeConfig.WakuV2Config.DiscoveryLimit,
-			DiscV5BootstrapNodes:    nodeConfig.ClusterConfig.DiscV5BootstrapNodes,
-			Nameserver:              nodeConfig.WakuV2Config.Nameserver,
-			EnableDiscV5:            nodeConfig.WakuV2Config.EnableDiscV5,
-			UDPPort:                 nodeConfig.WakuV2Config.UDPPort,
-			AutoUpdate:              nodeConfig.WakuV2Config.AutoUpdate,
-			DefaultShardPubsubTopic: shard.DefaultShardPubsubTopic(),
-			UseShardAsDefaultTopic:  nodeConfig.WakuV2Config.UseShardAsDefaultTopic,
-			TelemetryServerURL:      telemetryServerURL,
-			ClusterID:               nodeConfig.ClusterConfig.ClusterID,
+			MaxMessageSize:           wakucommon.DefaultMaxMessageSize,
+			Host:                     nodeConfig.WakuV2Config.Host,
+			Port:                     nodeConfig.WakuV2Config.Port,
+			LightClient:              nodeConfig.WakuV2Config.LightClient,
+			KeepAliveInterval:        nodeConfig.WakuV2Config.KeepAliveInterval,
+			Rendezvous:               nodeConfig.Rendezvous,
+			WakuNodes:                nodeConfig.ClusterConfig.WakuNodes,
+			EnablePeerExchangeServer: nodeConfig.WakuV2Config.PeerExchange,
+			EnableStore:              nodeConfig.WakuV2Config.EnableStore,
+			StoreCapacity:            nodeConfig.WakuV2Config.StoreCapacity,
+			StoreSeconds:             nodeConfig.WakuV2Config.StoreSeconds,
+			DiscoveryLimit:           nodeConfig.WakuV2Config.DiscoveryLimit,
+			DiscV5BootstrapNodes:     nodeConfig.ClusterConfig.DiscV5BootstrapNodes,
+			Nameserver:               nodeConfig.WakuV2Config.Nameserver,
+			EnableDiscV5:             nodeConfig.WakuV2Config.EnableDiscV5,
```

Comment:
Should this also be calculated? Previously it was `EnableDiscV5: !cfg.LightClient`